adidas-ascii-logo/README.md

```diff
@@ -0,0 +1,54 @@
+# adidas ASCII logo
+
+This challenge consists of implementing a function which, with the width of the stripe, returns the adidas logo using `@` character.
+
+_Extra:_ the goal is not only to return the adidas logo, but implementing the function with the less number of characters possible.
```

Comment:
`less` -> `least`

Reply:
Maybe we could rephrase:

`Extra challenge: will you be able to create the function with the least number of characters possible?`